tests/components/lyric/test_config_flow.py

```diff
@@ -0,0 +1,70 @@
+"""Test the Honeywell Lyric config flow."""
+from unittest.mock import patch
+
```

Comment:
While we don't require coverage on the rest, we do require 100% coverage on config flows

`pytest --cov=homeassistant/components/lyric/ --cov-report term-missing -- tests/components/lyric/test_*.py`